bin/bashcov

```diff
@@ -14,8 +14,15 @@ coverage = runner.result
 
 require "simplecov"
 
-SimpleCov.command_name Bashcov.fullname
+SimpleCov.command_name Bashcov.command_name
 SimpleCov.root Bashcov.root_directory
-SimpleCov.at_exit { SimpleCov::Result.new(coverage).format! }
+
+result = SimpleCov::Result.new(coverage)
+if SimpleCov.use_merging
```

Comment:
Maybe there should be a quick mention about this in the README?

Reply:
Agreed, and added :smiley: